### ISSUE TITLE
Discussion: Bad FPS

### ISSUE THREAD

**Issue by JJGadgets** [tostercx/GTAO_Booster_PoC#1]
this isn’t an issue but I figured you could probably give some insight: does the bad JSON parser also explain the bad FPS despite low system resources? Switching most graphics settings (reasonably, like not maxing out your grass & water quality for example) doesn’t change the FPS, and no CPU part (in terms of usage %) shows a bottleneck.

**Comment by tostercx** [comment]
> does the bad JSON parser also explain the bad FPS

Nope, unrelated. This is just when first starting up online mode.

**Comment by Tmp341** [comment]
> 
> 
> this isn’t an issue but I figured you could probably give some insight: does the bad JSON parser also explain the bad FPS despite low system resources? Switching most graphics settings (reasonably, like not maxing out your grass & water quality for example) doesn’t change the FPS, and no CPU part (in terms of usage %) shows a bottleneck.

@JJGadgets I've said this to some people, and they said their graphics improved somehow:

There is a file called **settings.xml**, which keeps your graphics settings etc in check. I believe, even if you change some settings ingame, it doesn't completely change all. While the game is closed, if you backup & delete that file, you might see some improvements. You can find that file under **Documents->Rockstar Games->GTAV**.

**Comment by deton24** [comment]
@Tmp341 So you basically mean that some of the graphics settings parameters are not being written correctly after changes in main menu, so you need to delete settings file in order to make the game changes those details at all since some of the details weren't changing correctly at all in some cases?

**Comment by Tmp341** [comment]
> 
> 
> @Tmp341 So you basically mean that some of the graphics settings parameters are not being written correctly after changes in main menu, so you need to delete settings file in order to make the game changes those details at all since some of the details weren't changing correctly at all in some cases?

That's what i'm suspecting. I believe, GTA doesn't detect all changes (and writes in the file). If you open that file with notepad, you can see lots of lines in there.

Because, some players were saying "i've maxed all settings, still feels medium level graphics". After doing this, all settings rewritten (and all settings return back to default values), some of them (graphic card name etc) recalculated after starting game.
